### PR TITLE
add dan4ik repository

### PR DESCRIPTION
**Pull request by dan4ik605743**
The following points apply when adding a new repository to repos.json

- 

- [x] I ran `./bin/nur format-manifest` after updating `repos.json` (We will use the same script in github actions to make sure we keep the format consistent)

- [x] By including this repository in NUR I give permission to license the
content under the MIT license.

Clarification where license should apply:
The license above does not apply to the packages built by the
Nix Packages collection, merely to the package descriptions (i.e., Nix
expressions, build scripts, etc.).  It also might not apply to patches
included in Nixpkgs, which may be derivative works of the packages to
which they apply. The aforementioned artifacts are all covered by the
licenses of the respective packages.

### CODE REVIEW

**Comment by dan4ik605743** [comment]
Accidentally closed, sorry)

**Comment by Mic92** [comment]
mhm. looks like there is still a diff in the repo.json.

**Comment by dan4ik605743** [comment]
Damn, it's weird. I don't understand what's wrong with him. I tried to edit from both github and wim. Still nothing ... What should I do?

**Comment by dan4ik605743** [comment]
image
Well, I kind of fixed it again. Let's see what happens

**Comment by dan4ik605743** [comment]
What do i do?

**Comment by dan4ik605743** [comment]
image
I did this in nvim.

File: repos.json [hunk 1]
```diff
@@ -117,6 +117,10 @@
             "submodules": true,
             "url": "https://github.com/cwyc/nur-packages"
         },
+        "dan4ik605743": {
```
Comment (Mic92) [inline repos.json:120]: The indentation was off.

Comment (dan4ik605743) [reply repos.json:120]: Did I do everything right?

**Comment by dan4ik605743** [comment]
image
image
2 screenshot in compton / default.nix
The package works fine on my system. Through callPackage. What's wrong with meta ???

**Comment by dan4ik605743** [comment]
> image
> image
> 2 screenshot in compton / default.nix
> The package works fine on my system. Through callPackage. What's wrong with meta ???

Apparently in localhost he is serous about it. But nixpkgs doesn't. This works if added to maintainers officially. I have no other explanation then .. Okay, now I'll try to remove the maintainer from all the packages.

**Comment by dan4ik605743** [comment]
image
Fixed the meta in builds.Try to push turnips

**Comment by dan4ik605743** [comment]
image
Great, I'm waiting for you to push your turnips

**Pull request merged**